handlers/photo-tactile-svg/tactile_svg.py

```diff
@@ -219,8 +223,8 @@ def handle():
 
     rendering = {
         "type_id": "ca.mcgill.a11y.image.renderer.TactileSVG",
-        "description": ("Tactile SVG of photo with possibly object detection "
-                        "and/or semantic segmentation outputs"),
+        "description": ("Tactile rendering of photo with " +
```

Comment:
Unsure if the names "semantic segmentation" and "object detection" are very recognizable to the average user!